mod-invoice-storage/schemas/invoice_line.json

```diff
@@ -0,0 +1,156 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "invoice line",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "UUID of this invoice line",
+      "type": "string",
+      "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+    },
+    "description": {
+      "description": "Title as capured in the Purchase Order Line. In the absence of a POL link this could be used to describe the POL",
+      "type": "string"
+    },
+    "poLineIds": {
+      "description": "List of UUID`s of the POL",
+      "id": "poLineIds",
+      "type": "array",
+      "items": {
+        "description": "UUID of the purchase order line associated with invoice",
+        "type": "string",
+        "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+      }
+    },
+    "invoiceLineNumber": {
+      "description": "Sequentially generated and not editable by the user.",
+      "type": "string",
+      "readonly": true
+    },
+    "invoiceLineStatus": {
+      "description": "Invoice line status",
+      "type": "string",
+      "enum": [
+        "Open",
+        "Reviewed",
+        "Approved",
+        "Paid",
+        "Cancelled",
+        "Error"
+      ]
+    },
+    "vendorRefNo": {
+      "description": "This is the number used to match POLs created by import to invoices created by import",
+      "type": "string"
+    },
+    "productId": {
+      "description": "product identifier",
+      "type": "string"
+    },
+    "productIdType": {
+      "description": "the type of id",
+      "type": "string",
+      "enum": [
```

Comment:
Probably better to move to separate file in common place and refer here and in `details.json` of the PO Line schema.